api/v1/testkube.yaml

```diff
@@ -502,6 +533,37 @@ paths:
                 items:
                   $ref: "#/components/schemas/Problem"
 
+  /test-with-executions/{id}:
```

Comment:
would be nice to have GraphQL for our API :)

Reply:
Yeah, was trying to keep things simple, but who knows)